src/justfile.rs

```diff
@@ -474,6 +474,7 @@ mod tests {
   run_error! {
```

Comment:
This test is unrelated to this PR, so should be reverted. This PR should add additional tests, not change existing tests.